<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSpinIter</class>
 <widget class="QWidget" name="MSpinIter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="tbNext">
     <property name="toolTip">
      <string>next</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="resfile.qrc">
       <normaloff>:/new/icons/icons/downarrow.png</normaloff>:/new/icons/icons/downarrow.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+PgDown</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbPrev">
     <property name="toolTip">
      <string>prev</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="resfile.qrc">
       <normaloff>:/new/icons/icons/uparrow.png</normaloff>:/new/icons/icons/uparrow.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+PgUp</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbLast">
     <property name="toolTip">
      <string>last</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="resfile.qrc">
       <normaloff>:/new/icons/icons/downarrowto.png</normaloff>:/new/icons/icons/downarrowto.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+End</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbFirst">
     <property name="toolTip">
      <string>first</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="resfile.qrc">
       <normaloff>:/new/icons/icons/uparrowto.png</normaloff>:/new/icons/icons/uparrowto.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+Home</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>tbNext</tabstop>
  <tabstop>tbPrev</tabstop>
  <tabstop>tbLast</tabstop>
  <tabstop>tbFirst</tabstop>
 </tabstops>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
